docs/reference/query-languages/esql/_snippets/commands/layout/stats-by.md

```diff
@@ -110,11 +110,24 @@ It’s also possible to group by multiple values:
 
 :::{include} ../examples/stats.csv-spec/statsGroupByMultipleValues.md
 :::
+
 If all the grouping keys are multivalued then the input row is in all groups:
 
 :::{include} ../examples/stats.csv-spec/multi-mv-group.md
 :::
 
+The input **ROW** is in all groups. The entire row. All the values. Even group
```

Comment:
OK, so you are really emphasising this! Probably necessary, since it is far from obvious.